railties/lib/rails/generators/rails/app/templates/app/channels/application_cable/connection.rb

```diff
@@ -1,4 +1,3 @@
-# Be sure to restart your server when you modify this file. Action Cable runs in a loop that does not support auto reloading.
```

Comment:
Not unless you've got a `use ActionDispatch::Reloader` (or whatever it is) in there as well. But I would think running a separate cable server in development mode would be a pretty unusual configuration -- certainly not something worthy of a top-of-file comment. At worst, it could merit a caveat where we describe how to set up the separate instance.